client/internal/httpsender_test.go

```diff
@@ -14,6 +14,36 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func TestHTTPSender_makeOneRequestRoundtrip(t *testing.T) {
```

Comment:
What does this test verify? I can't see any assertions.

Reply:
not sure how to test, as I see that function does not return anything. any suggestions on how it can be done?